config/crd/bases/control.k8ssandra.io_cassandratasks.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
Wrong controller-gen

Reply:
Your PR updated this to v0.10.0 dude: https://github.com/k8ssandra/cass-operator/pull/452/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R247

Reply:
Yes. It's supposed to be 0.10.0 (that PR was done 3 months ago). You're recreating these with 0.8.0.

Reply:
Oh, right, sorry! Fixed.

Those files get generated via kubebuilder don't they? I ran make controller-gen because I know that is local to the repo, but we use a system level kubebuilder in this repo don't we? Maybe we should be using a local one to avoid this problem.

Reply:
it's because the make targets use a local version of controller-gen which is under `./bin` and is only re-downloaded if it's not already in that folder. You may have had an older version there, which can be fixed by just deleting the binary and re-running the make target.

Reply:
Yeah, it turns out I'd only updated the one in k8ssandra-operator, not cass-operator. My bad, apologies.